File: Solutions/1029. Two City Scheduling/1029. Two City Scheduling.cpp
class Solution {
public:
    int twoCitySchedCost(vector<vector<int>>& costs) {
        vector<int> tmp;
        int n=costs.size()/2,res=0;
        for(auto& cost:costs){
            res+=cost[0];
            tmp.push_back(cost[1]-cost[0]);
        }
        sort(tmp.begin(),tmp.end());
        for(int i=0;i<n;++i){
            res+=tmp[i];
        }
        return res;
    }
};

File: Solutions/1029. Two City Scheduling/1029. Two City Scheduling.go
func twoCitySchedCost(costs [][]int) int {
    var tmp []int
    n,res:=len(costs)/2,0
    for _,cost:=range(costs){
        res+=cost[0]
        tmp=append(tmp,cost[1]-cost[0])
    }
    sort.Ints(tmp)
    for i:=0;i<n;i+=1{
        res+=tmp[i]
    }
    return res
}